auth/src/main/res/values-v35/styles.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="ResourceName">
+
+    <!-- Override the base FirebaseUI theme for edge-to-edge support -->
+    <style name="FirebaseUI" parent="Theme.MaterialComponents.DayNight.DarkActionBar">
+        <item name="android:windowOptOutEdgeToEdgeEnforcement">true</item>
```

Comment:
We shouldn't rely on `windowOptOutEdgeToEdgeEnforcement` as that's going away in Android 16: https://developer.android.com/about/versions/16/behavior-changes-16#edge-to-edge